packages/salesforcedx-sobjects-faux-generator/test/integration/sObjectDescribe.test.ts

```diff
@@ -93,6 +101,26 @@ describe('Fetch sObjects', function() {
     expect(customField.name).to.be.equal('MyCustomField__c');
   });
 
+  it('Should be able to call describeSObjectBatch on custom objects', async function() {
+    const cmdOutput = await sobjectdescribe.describeSObjectBatch(
+      process.cwd(),
+      [CUSTOM_OBJECT_NAME, CUSTOM_OBJECT2, CUSTOM_OBJECT3],
+      0,
+      username
+    );
+    expect(cmdOutput[0].name).to.be.equal(CUSTOM_OBJECT_NAME);
+    expect(cmdOutput[0].custom).to.be.true;
+    expect(cmdOutput[0].fields.length).to.be.least(9);
```

Comment:
No magic numbers.

Reply:
Are you referring to the 9 fields?   Winston put that in earlier and I was just following it.   Do you want it to be a constant instead with a name?